index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+       
+
+<div class="container"><header class="header"> 
+  <h1 id="title"></h1>
+
+  <p id="description">Thank you for taking the time to help us improve the platform</p>
+</header>
+  <form id="survey-form">
+    <div class="form-group">
+      <label id="name-label" for ="name">Name</label>
+      <br>
+      <input
+      type="text"
+      name="name"
+      placeholder="Enter your name"
+      required
+    </div>
+
+    <div class="form-group">
+      <label id="email-label" for ="email">Email</label>
+      <br>
+      <input
+      type="email"
+      name="email"
+      placeholder="Enter your Email"
+      required
+    </div>
+
+    <div class="form-group">
+      <label id="age-label" for ="age">Age</label>
+      <br>
+      <input
+      type="age"
+      name="age"
+      placeholder="Age"
+      required
+    </div>
+
```

Comment:
- [ ] All of its attributes are optional, but it's standard practice to always set at least the **[action]** and the **[method]** to the forms.